oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/endpoint/WebClientReactiveClientCredentialsTokenResponseClient.java

```diff
@@ -101,4 +104,9 @@
 		}
 		return body;
 	}
+
+	public void setWebClient(WebClient webClient){
+		Assert.notNull(webClient, "Custom WebClient must not be null");
```

Comment:
To follow the conventions of Spring Security, please change to "webClient cannot be null".

Reply:
Gotcha, I was wondering if there was a convention. I think I pulled that from somewhere else in the code.